src/lib/pages/my-published-modules/components/MyPublishedModulesTable.tsx

```diff
@@ -0,0 +1,86 @@
+import { useMemo, useState } from "react";
+
+import { useCurrentChain } from "lib/app-provider";
+import InputWithIcon from "lib/components/InputWithIcon";
+import { DisconnectedState, EmptyState } from "lib/components/state";
+import { ModulesTable } from "lib/components/table";
+import { useFormatAddresses } from "lib/hooks/useFormatAddresses";
+import { useAPIAccountModules } from "lib/services/move/moduleService";
+import type { HumanAddr } from "lib/types";
+
+export const MyPublishedModulesTable = () => {
+  const [keyword, setKeyword] = useState("");
+  const { address } = useCurrentChain();
+
+  const {
+    data: modulesData,
+    isFetching: isModulesLoading,
+    error,
+  } = useAPIAccountModules(address as HumanAddr);
+
+  const formatAddresses = useFormatAddresses();
+  const mappedModules = modulesData?.map((module) => {
+    return {
+      address: formatAddresses(module.address).address,
+      name: module.moduleName,
+      functions: {
+        view: module.viewFunctions.length,
+        execute: module.executeFunctions.length,
+      },
+    };
+  });
+
+  const filteredPublishedModules = useMemo(() => {
+    if (!keyword) return mappedModules;
+
+    return mappedModules?.filter(
+      (module) => module.name?.toLowerCase().includes(keyword.toLowerCase())
+    );
+  }, [keyword, mappedModules]);
+
+  const emptyState = () => {
+    if (error)
+      return (
+        <EmptyState
+          withBorder
+          imageVariant="not-found"
+          message="There is an error during fetching recent modules."
+        />
```

Comment:
use `<ErrorFetching message="There is an error during fetching recent modules." />`